README.md

```diff
@@ -19,9 +19,44 @@ github "Dronecode/DronecodeSDK-Swift" ~> 0.3.0
 And then get the framework using:
 
 ```shell
+rm -rf Carthage ~/Library/Caches/org.carthage.CarthageKit ~/Library/Caches/carthage
```

Comment:
I would note that this cleaning the caches is sometimes needed if something goes wrong. I would probably not advise to always delete it no matter what.

Reply:
Or maybe add that to a "Troubleshooting" section for Carthage?

Reply:
I like that.